helm/crds/elasticache.services.k8s.aws_cacheclusters.yaml

```diff
@@ -837,10 +837,6 @@ spec:
                   numCacheNodes:
                     format: int64
                     type: integer
-                  transitEncryptionEnabled:
-                    type: boolean
-                  transitEncryptionMode:
-                    type: string
                 type: object
               replicationGroupLogDeliveryEnabled:
                 description: |-
```

Comment:
Are we removing any fields from the cacheclusters CRD? how come the respective Go file isn't getting modified?